app/src/main/java/com/google/droidjump/AchievementDetailsFragment.java

```diff
@@ -89,5 +101,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         }
         return rootView;
     }
+
+    private int getPercentCountOfProgress(Achievement achievement) {
```

Comment:
Do we really need to user percentages, why not to show steps directly? It can be misleading if your achievement is "collect all 37 coins" - why would we show percentages instead of a clear progress like "10/37"?

Reply:
I did the same as in the standard UI. In the image of progress bar, percentages look more logical. To be able to see clear progress, I can add text with this information to AchievementDetails fragment.

Reply:
Done